README.md

```diff
@@ -4,33 +4,64 @@ Software Engineering Day1 Assignment
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
-
+Software engineering is the application of engineering principles to design,develop,test and deploy softwate systems.
+It helps to create software application in various aspects of modern life,such as commerce,communication,entertainmentvand health care.
 
 Identify and describe at least three key milestones in the evolution of software engineering.
-
+Development of programming languages
+Establishment of software engineering as a discipline in the 1960's
+The advent of structured programming in the 1970's
 
 List and briefly explain the phases of the Software Development Life Cycle.
-
+1. Planning
+2. Analysis
+3. Design
```

Comment:
A softwae developer Writes, tests, and maintains code to build software applications, implements features, fixes bugs, and collaborates with the team to meet project goals.
A Qualiy assurance engineer Ensures the software meets quality standards by designing and executing tests, identifying defects, and working with developers to resolve issues, focusing on delivering a reliable product.
A project manager Oversees the project's progress, manages timelines, resources, and budgets, coordinates between team members, and ensures the project meets client requirements and deadlines.